srml/support/src/dispatch.rs

```diff
@@ -867,9 +876,31 @@ macro_rules! decl_module {
 		$vis fn $name(
 			$origin: $origin_ty $(, $param: $param_ty )*
 		) -> $crate::dispatch::Result {
-			{ $( $impl )* }
-			// May be unreachable.
-			Ok(())
+			#[cfg(feature = "std")]
```

Comment:
why are we not gating this in another feature like `#[cfg(feature = "profile")]`?

Reply:
as of now, I think these extra timing logics are exected all the time on a client with native runtime and sending a lot of data to telemetry; do we want this? (maybe we do and I don't know)

Reply:
wrt feature - good question, I thought to do it this way, but it's possible also to use feature gate. Will answer q about telemetry below...